src/main/resources/templates/image-resizer.html

```diff
@@ -0,0 +1,85 @@
+﻿<!doctype html>
+<html xmlns:th="http://www.thymeleaf.org" th:replace="~{util/layout::base-page('Resize Image', ~{::content})}">
+<meta charset="UTF-8">
+<body>
+<main th:fragment="content">
+  <h2>Image Resizer Tool</h2>
+  
+  <div class="tool-description">
+    <p>Upload an image to resize it to your desired dimensions. Maximum file size is 10MB.</p>
+    <p>Supported formats: JPEG, PNG, GIF, BMP</p>
+  </div>
+  
+  <form action="/tools/image-resizer/process" method="post" enctype="multipart/form-data" id="resizeForm">
+    <div class="form-group">
+      <label for="imageFile">Choose image:</label>
+      <input type="file" id="imageFile" name="imageFile" accept="image/*" required>
+      <small>Max file size: 10MB</small>
+    </div>
+
+    <div class="form-group">
+      <label for="width">New width (px):</label>
+      <input type="number" id="width" name="width" min="1" max="5000" value="800" required>
+    </div>
+
+    <div class="form-group">
+      <label for="height">New height (px):</label>
+      <input type="number" id="height" name="height" min="1" max="5000" value="600" required>
+    </div>
+
+    <button type="submit" class="btn btn-primary" id="resizeButton">Resize</button>
+  </form>
+  
+  <div id="loadingIndicator" style="display: none;" class="mt-3">
+    <div class="d-flex align-items-center">
+      <strong>Processing image...</strong>
+      <div class="spinner-border ml-auto" role="status" aria-hidden="true"></div>
```

Comment:
This doesn't do anything as we don't have the appropriate styles set up. Remove of style this.